src/app/components/Parallels/ParallelsFields.jsx

```diff
@@ -0,0 +1,49 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { extractParallels } from '../../utils/bibDetailsUtils';
+
+const ParallelsFields = ({ field, content = '', fieldIndex = 0, bib }) => {
+  if (!content) return null;
+
+  const parallel = extractParallels(bib, field);
+
+  return (
+    <>
+      {(parallel &&
+        parallel[fieldIndex] &&
+        parallel[fieldIndex]
+          .map((value, idx) => {
+            if (!value) return null;
+
+            return (
+              <span
+                key={`${idx}_${fieldIndex}_${value}`}
+                dir={unicodeDirection(value)}
+                style={{ display: 'block' }}
+              >
+                {value}
+              </span>
+            );
+          })
+          .filter(Boolean)) || (
+        <span dir={unicodeDirection(content)} style={{ display: 'block' }}>
+          {content}
+        </span>
+      )}
+    </>
+  );
+};
+
+export default ParallelsFields;
+
+ParallelsFields.propTypes = {
+  content: PropTypes.string,
+  field: PropTypes.string,
+  fieldIndex: PropTypes.number,
+  headingLevel: PropTypes.number,
+  bib: PropTypes.object,
+};
```

Comment:
> What is headingLevel?

Left over. Can be removed.

> Still think this is a confusing component

Confusing? Seems like you get it! :) Kidding, it's confusing. The reason for this is because of (dun dun dun) Notes and/or something else. 

Due to the comment below ↓, Parallels may need to be refactored so it's likely this component will be changed or completely removed.

> I see we're abandoning transforming the bib object and going back to triggering parallel extraction.

Correct! I felt there was a better solution than adding it to the bib object.